src/libstd/sys/unix/fs.rs

```diff
@@ -25,15 +25,21 @@ use sys::time::SystemTime;
 use sys::{cvt, cvt_r};
 use sys_common::{AsInner, FromInner};
 
-#[cfg(target_os = "linux")]
+#[cfg(any(target_os = "linux", target_env = "musl", target_os = "emscripten"))]
```

Comment:
Because MUSL implies `target_os = "linux"` I think that clause can be removed from this and the bits below